install_config/install/advanced_install.adoc

```diff
@@ -681,6 +681,154 @@ openshift_examples_modify_imagestreams=true
+link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html/container-native_storage_for_openshift_container_platform_3.4/[Red Hat Container Native Storage (CNS)] to create containerized GlusterFS storage
+pods that back the registry.
+endif::[]
+ifdef::openshift-origin[]
+See link:https://github.com/gluster/gluster-kubernetes[Running Containerized GlusterFS in Kubernetes] for additional information on containerized storage
+using GlusterFS.
+endif::[]
+
+After successful installation, all required components to start using and
+provisioning storage will exist, including `Endpoints`, `Services` and a
+`StorageClass`.
+
+To configure containerized GlusterFS persistent storage:
+
+. Specific prerequisites and host preparation are required to run this deployment.
+See
+xref:../../install_config/install/prerequisites.adoc#prereq-containerized-glusterfs-considerations[Containerized GlusterFS Considerations] and ensure the steps have been satisfied.
+
+. In your inventory file, add `glusterfs` to the `OSEv3:children` section to enable the `[glusterfs]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs
+----
+
+. (Optional) Include any of the following role variables in the `[OSEv3:vars]`
+section you wish to change:
++
+----
+[OSEv3:vars]
+openshift_storage_glusterfs_namespace=glusterfs <1>
+openshift_storage_glusterfs_name=storage <2>
+----
+<1> The project (namespace) to host the storage pods. Defaults to `glusterfs`.
+<2> A name to identify the GlusterFS cluster, which will be used in resource names.
+Defaults to `storage`.
+
+. Add a `[glusterfs]` section with entries for each storage node that will host
+the GlusterFS storage and include the `glusterfs_devices` list parameter in the form:
++
+----
+<hostname_or_ip> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs] 
+192.168.10.11 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+192.168.10.12 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+192.168.10.13 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+----
++
+The `glusterfs_devices` list of raw block devices will be completely managed as
+part of a GlusterFS cluster. There must be at least one device listed. Each
+device must be bare, with no partitions or LVM PVs.
+
+. Additional steps must be run manually after the cluster installation has
+completed. Continue with the rest of the preparation of your inventory file,
+then see xref:running-the-advanced-installation[Running the Advanced Installation] for the additional steps to complete the configuration of
+containerized GlusterFS persistent storage.
+
+[[advanced-install-configuring-openshift-container-registry]]
+=== Configuring the OpenShift Container Registry
+
+Additional configuration options are available at installation time for the
+xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[OpenShift Container Registry].
```

Comment:
I would recommend consolidating on "Integrated OpenShift Container Registry" everywhere. This is because all this GlusterFS registry stuff currently only applies to containerized/hosted registries, while there are also non-containerized deployments of an OpenShift Container Registry. If anything, I might suggest both the above and maybe "container registry" or just "registry" shorthand in cases where the longer name has already been used in the same sentence/paragraph.